mailers/password-mailer/password-mailer.go

```diff
@@ -1,4 +1,4 @@
-package password_mailer
+package passwordmailer
 
```

Comment:
__[golint]__
* [golint] comment on exported function Reset should be of the form "Reset ..."